axlearn/common/attention_bias.py

```diff
@@ -701,6 +701,11 @@ def sliding_window_causal_mask(sliding_window_size: int) -> MaskFn:
     """Returns a causal MaskFn for sliding window attentions of a given window size.
 
     Implements the `MaskFn` protocol.
+
+    Note: Setting sliding_window_size = 8 results in a window size of 9, including itself.
```

Comment:
nit: Maybe explicitly note that it attends to itself and sliding_window_size tokens on the left. "window size" is not a well defined term.

Reply:
I see. Changed to
```
    Note: Setting sliding_window_size = 8 results in attending to 9 tokens - it attends to itself
    and sliding_window_size tokens to the left.
```